banyand/measure/service.go

```diff
@@ -78,6 +79,8 @@ func (s *service) FlagSet() *run.FlagSet {
 	flagS := run.NewFlagSet("storage")
 	flagS.StringVar(&s.root, "measure-root-path", "/tmp", "the root path of database")
 	flagS.DurationVar(&s.option.flushTimeout, "measure-flush-timeout", defaultFlushTimeout, "the memory data timeout of measure")
+	s.option.maxFanOutSize = math.MaxInt64
```

Comment:
Move the `maxFanoutSize` into `s`. Initialize the `mergePolicy` in `PreRun`. Finally, assign the policy to the `s.option`

Reply:
Fixed. Pls check the new impl